common/src/main/java/org/apache/omid/NetworkUtils.java

```diff
@@ -17,15 +17,19 @@
  */
 package org.apache.omid;
 
+import org.slf4j.Logger;
```

Comment:
Please see https://omid.incubator.apache.org/coding-guide-and-style.html on import ordering convetions

Reply:
will copy that

Reply:
@stoty How about making checking for violations as part of the build? It can include import rule.

Reply:
That would need to be smart, as I don't expect that the current code base is fully compliant either.
Yetus has this functionality, but we aren't using that for Omid.

In any case, that should be handled in a different ticket.

Reply:
> That would need to be smart, as I don't expect that the current code base is fully compliant either.
Yetus has this functionality, but we aren't using that for Omid.

sure. that should be addressed by other issue(s). I'm going to open tickets to include following tasks:

1. add checkstyle:check to validate phase (violations are viewed as warning rather than error)
1. fix all warnings of common module
1. fix all warnings of ...
1. fail build if there is any violation

@stoty WDYT?

Reply:
I have recently added Yetus support for phoenix and phoenix-connectors, you may want to do the same for Omid, so that the Phoenix projects have similar test setup. Apart from checkstyle, spotbugs is also a highly recommended to tool for this, for the possible logic errors.

I am not sure what you mean by warnings of common module.

Generally, fixing up the code to address static analysis / checkstyle warnings is very useful up to a point.
If you have the time and motivation to work on this, then go ahead by all means!